dev/script/generate-config/service-config.js

```diff
@@ -94,6 +94,11 @@ function getKahunaConfig(config){
         |  }
         |]`;
 
+    const permissionsConfig = stripMargin`
+      |usePermissionsFilter=true
+      |permissionsDefault="allPermissions"
+      `;
```

Comment:
I know this is for the default option on the image permissions filter in the UI, but `permissionsDefault="allPermissions"` in the config looked like a security bug at first glance (sounds like "give the user all the permissions unless told otherwise") - could we add a comment here to clarify what this config controls for future reference?